### ISSUE TITLE
default exposure blending not working

### ISSUE THREAD

**Issue by TurboGit** [darktable-org/darktable#13500]
**Describe the bug/issue**

Using blending (drawn or parametric masks) do not work on a scene referred image default exposure.

**To Reproduce**
1. Select scene referred workflow
1. Discard history of a picture
1. Go to 'darkroom'
2. Select 'exposure' module, see that default has been set to 0.7EV
3. Add a drawn mask, see nothing happens
4. Add a parametric mask, see nothing happens

**Expected behavior**

Only the mask should have the 0.7EV exposure.

**Which commit introduced the error**

Don't know yet.

**Platform**

* darktable version : master
* OS                : GNU/Linux
* Linux - Distro    : Debian/sid

**Comment by TurboGit** [comment]
Can anyone reproduce? TIA.

**Comment by TurboGit** [comment]
I'm the winner :)

```
commit a08bd88743abe8f719efa07d81a4b7978a0b7bf4
Author: Pascal Obry <pascal@obry.net>
Date:   Wed Dec 14 21:14:15 2022 +0100

    develop: Add support for multiple auto-presets of same operation.
    
    The SQL statement is changed to support multiple presets. Here we order each
    multiple instance with a proper multi_priority (ordered from 0 to n). Also
    we give to the history the name of the preset multi_name or if empty the
    name of the preset itself.
    
    After that we ensure that we create the multiple entries for the iop-order.
    
    Fixes #12853.
```

**Comment by TurboGit** [comment]
Hum... In fact everything is working as expected... The default for scene-referred is 0.7 and the compensation for my picture is 0.7... So a mask does nothing indeed as 0.7 is the neutral value!
